src/lib.rs

```diff
@@ -228,7 +228,8 @@ mod macros;
 // refcount does not necessarily monotonically decrease at this point, as new
 // log calls still increment and decrement it, but the interval in between is
 // small enough that the wait is really just for the active log calls to finish.
-static mut LOGGER: Option<*const Log> = None;
+
+static mut LOGGER: *const Log = &NopLogger;
```

Comment:
Ideally we'd just set this to a "null" pointer, but I'm not sure if you can assemble one of those for a fat pointer in a const expr.